resources/server/src/landingpage/osv-scanner.toml

```diff
@@ -0,0 +1,23 @@
+[[IgnoredVulns]]
+id = "GHSA-8jmw-wjr8-2x66"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
```

Comment:
GHSA-8jmw-wjr8-2x66 is about git-clone, which is not used by the JS application, I have no idea where this vulnerability comes from.

Reply:
After looking through the yarn.lock file under the landingpage directory. You've got git-clone listed as a dependency for the download-git-repo (line 3837) and listed on it's own (line 4707).

Turns out your initial reasoning that this is part of the landing page application was right.